packages/backend/services/propertyService.js

```diff
@@ -36,26 +45,55 @@ export const getPropertiesWithinPolygonsCoordinates = (polygonsData) => {
       });
     }
   });
+  propertiesWithinPolygons.forEach((property) => {
+    calculateTravelToWork(property);
+  });
   return [...new Set(propertiesWithinPolygons.flat())];
 };
 
-const getDistance = async (startCoordinates, endCoordinates, methodOfTravel) => {
+const getDirections = async (
+  startCoordinates,
+  endCoordinates,
+  methodOfTravel
+) => {
   const token = `?access_token=${process.env.MAPBOX_TOKEN}`;
-
   const { data } = await axios.get(
-    `https://api.mapbox.com/directions/v5/mapbox/${methodOfTravel}/${startCoordinates}${endCoordinates}?access_token=${token}`
+    `https://api.mapbox.com/directions/v5/mapbox/${methodOfTravel}/${startCoordinates};${endCoordinates}${token}`
   );
+  return data.routes;
+};
 
-  data.routes.forEach((route) => {
-    route.durationByTrain = calculateTrainDuration(route.distance, route.duration);
-    route.durationByBus = calculateBusDuration(route.distance, route.duration);
-  });
+const getNearestPointOfInterest = async (pointOfInterest, coordinates) => {
+  const resp = await axios.get(
+    `https://api.mapbox.com/geocoding/v5/mapbox.places/${pointOfInterest}.json?type=poi&proximity=${coordinates}&access_token=${token}`
+  );
+  const data = [
+    resp.data.features[0].properties.address,
+    resp.data.features[0].context[0].text,
+    resp.data.features[0].geometry.coordinates,
+  ];
   return data;
 };
 
-//TOTO what does this???
-const getNearestWhat = async (pointOfInterest, coordinates) => {
-  const url = `https://api.mapbox.com/geocoding/v5/mapbox.places/${pointOfInterest}.json?type=poi&proximity=${coordinates}&access_token=${token}`;
-  const resp = await axios.get(url);
-  return resp.data;
+const calculateTravelToWork = async (property) => {
+  const endCoordinates = AND_DIGITAL_COORDINATES;
+
+  const startCoordinates = [
+    property.details.coordinates[1],
+    property.details.coordinates[0],
+  ];
+
+  for (let i = 0; i < METHOD_OF_TRAVEL.length; i++) {
```

Comment:
a for of should work aswell here

Reply:
What are the advantages? Is it worth changing or just letting me know that there are other options?

Reply:
Looked into this and like how clean it is so just swapping it over